api/src/pcapi/routes/backoffice/offerers/offerer_blueprint.py

```diff
@@ -1015,17 +1012,44 @@ def update_individual_subscription(offerer_id: int) -> utils.BackofficeResponse:
         "isCertificateValid": form.is_certificate_valid.data,
     }
 
-    if offerer.individualSubscription:
-        offerers_models.IndividualOffererSubscription.query.filter_by(offererId=offerer_id).update(
-            data, synchronize_session=False
-        )
-    else:
-        db.session.add(offerers_models.IndividualOffererSubscription(offererId=offerer_id, **data))
+    offerers_models.IndividualOffererSubscription.query.filter_by(offererId=offerer_id).update(
+        data, synchronize_session=False
+    )
     db.session.commit()
 
     return _self_redirect(offerer_id, active_tab="subscription")
 
 
+@offerer_blueprint.route("/create-individual-subscription", methods=["GET"])
+@utils.permission_required(perm_models.Permissions.VALIDATE_OFFERER)
+def create_individual_subscription(offerer_id: int) -> utils.BackofficeResponse:
+    offerer = (
+        offerers_models.Offerer.query.options(
+            sa.orm.joinedload(offerers_models.Offerer.UserOfferers)
+            .joinedload(offerers_models.UserOfferer.user)
+            .load_only(users_models.User.email),
+            sa.orm.joinedload(offerers_models.Offerer.individualSubscription),
+        )
+        .filter(offerers_models.Offerer.id == offerer_id)
+        .one_or_none()
+    )
+
+    if not offerer:
+        raise NotFound()
+
+    if not offerer.individualSubscription:
```

Comment:
Dans les données existantes, il n'est pas impossible d'avoir un objet `IndividualOffererSubscription` sans avoir envoyé l'email, car rien n'empêchait de remplir un champ sans cocher l'envoi d'email. Dans ce cas, rien ne sera envoyé ci-dessous. Est-ce un comportement à gérer ?

Cette requête sur la base de production me renvoie 61 lignes, dont une partie avec aucun champ coché/rempli : 
`select * from individual_offerer_subscription where "isEmailSent" is false;`

D'ailleurs, dans ce cas on se retrouve dans le code plus haut (et le jinja plus bas) à afficher le formulaire, qui ne permet plus de cocher l'email envoyé, mais ne permet pas de l'envoyer. À clarifier.